plants.rb

```diff
@@ -1,13 +1,15 @@
 require 'sinatra'
 require 'faraday'
 require 'fast_jsonapi'
-
 require './poros/plant'
```

Comment:
Are we going to need the poro?

Reply:
ah you're right! I don't think so since we aren't serializing anything?

Reply:
Removed!